app/helpers/blacklight/blacklight_helper_behavior.rb

```diff
@@ -253,7 +261,7 @@ def with_format(format)
   # Should we render a grouped response (because the response
   # contains a grouped response instead of the normal response)
   def render_grouped_response? response = @response
-    response.grouped?
+    response&.grouped?
```

Comment:
why do we have to do a nil check?

Reply:
Mostly for testing convenience. The awful  mechanics for stubbing `@response` in the view context for testing a  component when... we don't care... seemed fishy.

Reply:
Can we call that out here in a comment?